packages/ra-ui-materialui/src/detail/SimpleShowLayout.spec.js

```diff
@@ -12,7 +12,7 @@ describe('<SimpleShowLayout />', () => {
                 <TextField source="bar" />
             </SimpleShowLayout>
         );
-        const inputs = wrapper.find('pure(TextField)');
+        const inputs = wrapper.find('EnhancedTextField');
```

Comment:
By setting the displayName on the `EnhancedTextField`, we can avoid testing implementation details here